Rakefile

```diff
@@ -22,9 +22,10 @@ YARD::Rake::YardocTask.new do |t|
 end
 
 RuboCop::RakeTask.new
-
-RSpec::Core::RakeTask.new(:spec) do |r|
-  r.pattern = FileList['**/**/*_spec.rb']
+FileList['**/**/*_spec.rb'].each do |f|
```

Comment:
interesting, I am fine with this change but we really should look into it as this increases the amount of time it takes for CI to run.